apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -25,8 +35,8 @@
                             "description" :
                             "The peer address that the node will present itself with. If port part given, it must match 'port' user config.",
```

Comment:
@velzevur This "If port part given" is the portion that I was mentioning may need update (and potentially delete the port separate config at line 41 below...).

Reply:
The quickest way to fix the problem was to make the code in `aehttp_app` match specifically on the external port, but it does seem silly to make the port part mandatory, and then have an optional "port" property besides it. Better then to specify only the host part, if needed, then optionally a port separately. Especially since the host part is used also for the internal_peer_address.